bindings/python/sample.py

```diff
@@ -22,11 +22,11 @@ def test_ks(arch, mode, code, syntax=0):
 
 # test symbol resolver
 def test_sym_resolver():
-    def sym_resolver(symbol, value):
+    def sym_resolver(symbol, p_value):
         # is this the missing symbol we want to handle?
         if symbol == "_l1":
             # put value of this symbol in @value
-            value = 0x1002
+            p_value.contents.value = 0x1002
```

Comment:
ok, i implemented a wrapper inside the Python binding, see the new branch https://github.com/keystone-engine/keystone/tree/test

now bindings/python/sample.py works, but somehow both `suite/regress/x64_sym_resolver.py` & `suite/regress/x86_call_ptr_sym.py` crash on my Mac. not sure why yet. need to go, so if you find out, let me know.